schedule.go

```diff
@@ -0,0 +1,187 @@
+package helix
+
+// GetScheduleParams are the parameters for GetSchedule
+type GetScheduleParams struct {
+	BroadcasterID string `json:"broadcaster_id"`
+	ID            string `json:"id"`
+	StartTime     string `json:"start_time"`
```

Comment:
`StartTime` should be of type [`Time`](https://github.com/nicklaw5/helix/blob/main/time.go#L24), rather than `string`.